# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../keepalived/keepalived-1.2.2-ip_vs.h-pathfix.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix the path where configure is searching for ip_vs.h

--- keepalived-1.2.2/configure.in.orig	2012-05-09 16:44:25.195527122 +0200
+++ keepalived-1.2.2/configure.in	2012-05-09 16:45:05.336310210 +0200
@@ -124,7 +124,7 @@
 dnl ----[ Checks for LVS and VRRP support ]----
 IPVS_SUPPORT="_WITHOUT_LVS_"
 if test "$enable_lvs" != "no"; then
-  AC_CHECK_HEADER([net/ip_vs.h],
+  AC_CHECK_HEADER([linux/ip_vs.h],
     [IPVS_SUPPORT="_WITH_LVS_"],
     [
       IPVS_SUPPORT="_WITHOUT_LVS_"
